Jenkinsfile

```diff
@@ -1090,7 +1090,7 @@ pipeline {
                                        inst_repos: el7_component_repos + ' ' + component_repos(),
                                        inst_rpms: 'gotestsum openmpi3 hwloc-devel argobots ' +
                                                   "fuse3-libs fuse3 " +
-                                                  'libisa-l-devel libpmem libpmemobj protobuf-c ' +
+                                                  'libisa-l-devel libisa-l_crypto-devel libpmem libpmemobj protobuf-c ' +
```

Comment:
Did you add this because the unit test broke without it?  `-devel` packages added here are generally only needed for the GO unit tests.

Reply:
mostly, I added it because I was duplicating what was done for the isa-l. I will remove and try without.